index.md

```diff
@@ -17,6 +17,9 @@ excerpt: "The Belgium Azure User Group focuses on knowledge sharing and networki
 **AZUG and Covid-19** - After careful consideration, we've decided to make the next AZUG meeting virtual, and cancel Global Azure Bootcamp in Belgium. [Read more!](https://www.azug.be/events/2020/03/13/azug-and-covid-19){: .btn .btn--warning}
 {: .notice--warning}
 
+**July 7th 2020** - 7 habits every Azure Admin must have by **Wim Matthyssen** and **Karel De Winter** [Read more!](https://www.azug.be/events/2020/07/07/2020-07-07---7-habits-every-azure-admin-must-have){: .btn .btn--warning}
+{: .notice--warning}
```

Comment:
Let's use info instead of warning

```suggestion
{: .notice--info}
```

![image](https://user-images.githubusercontent.com/4345663/84407501-39a5b400-ac0b-11ea-9459-a094823b7902.png)